src/js/utils.js

```diff
@@ -2,6 +2,17 @@
 
 /* global convertDate, stripHtmlTags, htmlDecode, chunkArray, escapeSpecialChars */
 
+/*
+ * Remove soft hyphens and replace &#8217; with apostrophe.
+ * @param {String} webpageString - A string with one or more soft hyphens or &#8217;.
+ * @return The string with soft hyphens removed and &#8217; replaced with an apostrophe.
+ */
+// eslint-disable-next-line
+metaTitle = function (webpageString) {
+	webpageString = webpageString.replace(/&#8217;/g, "’");
```

Comment:
It would be better to use a logic like:

```
const toReplace = {
    "&#8217;": "’",
    "&shy;": ""
};

// Iterate thru toReplace object to replace every match.
```

When a new replacement needs to be added, extending the data in `toReplace` will be enough rather than copying the `.replace()` code over and over.